mitosheet/mitosheet/code_chunks/excel_range_import_code_chunk.py

```diff
@@ -0,0 +1,46 @@
+
+#!/usr/bin/env python
+# coding: utf-8
+
+# Copyright (c) Saga Inc.
+# Distributed under the terms of the GPL License.
+from typing import Any, List, Optional
+from mitosheet.code_chunks.code_chunk import CodeChunk
+from mitosheet.excel_utils import get_column_from_column_index, get_row_and_col_indexes_from_range
+from mitosheet.types import ColumnID
+from mitosheet.state import State
+
+class ExcelRangeImportCodeChunk(CodeChunk):
+
+    def __init__(self, prev_state: State, post_state: State, file_name: str, sheet_name: str, range_imports: Any):
+        super().__init__(prev_state, post_state, {}, {})
+        self.file_name = file_name
+        self.sheet_name = sheet_name
+        self.range_imports = range_imports
+
+    def get_display_name(self) -> str:
+        return 'Excel Range Import'
+    
+    def get_description_comment(self) -> str:
+        
+        return f"Imported {len(self.range_imports)} dataframes from {self.sheet_name} in {self.file_name}"
+
+    def get_code(self) -> List[str]:
+        code = []
```

Comment:
Until we do the code optimization, this should be `code = ['import pandas as pd']`